README.md

```diff
@@ -321,66 +321,100 @@ Fidel.setOptions({
+
+```javascript
+Fidel.setOptions({
   programName: 'Your Program Name',
 });
 ```
 
-The default placeholder value for programName is `our` (in English). The `termsConditionsUrl` is mandatory in this case. If you don't set a privacy policy URL (which is different than the terms & conditions URL), the corresponding wording will not be displayed.
+#### Text behaviour for privacy policy
+
+Notice the following excerpt from the consent text above: _in accordance with the ```Cashback Inc``` Terms and Conditions, ```Cashback Inc``` privacy policy and Fidel’s Privacy Policy._ If you set a ```privacyUrl```, this is the text that will be displayed, along with a hyperlink set on *privacy policy*.
+
+If you do not set a ```privacyUrl```, the text will become _in accordance with the ```Cashback Inc``` Terms and Conditions and Fidel’s Privacy Policy._
+
+
+### Consent text for the rest of the world
+
+When you set United Kingdom, Ireland, Japan or Sweden as the default country or the user selects one of these countries from the lost, a specific consent text will be applied.
+
+The following would be an example Terms & Conditions text for ```Cashback Inc``` (an example company):
 
-For USA & Canada, the following would be an example Terms & Conditions text, for Cashback Inc (an example company):
+*I authorise ```card_scheme_name``` to monitor my payment card to identify transactions that qualify for a reward and for ```card_scheme_name``` to share such information with ```Cashback Inc```, to enable my card linked offers and target offers that may be of interest to me. For information about ```Cashback Inc``` privacy practices, please see the privacy policy. You may opt-out of transaction monitoring on the payment card you entered at any time by ```deleteInstructions```.*
 
-*By submitting your card information and checking this box, you authorize Visa to monitor and share transaction data with Fidel (our service provider) to participate in  program. You also acknowledge and agree that Fidel may share certain details of your qualifying transactions with Cashback Inc to enable your participation in  program and for other purposes in accordance with the Cashback Inc Terms and Conditions, Cashback Inc privacy policy and Fidel’s Privacy Policy. You may opt-out of transaction monitoring on the linked card at any time by contacting support.*
+#### Text behaviour for privacy policy
 
-For the rest of the world:
+Notice the following excerpt from the consent text above: _...may be of interest to me. For information about ```Cashback Inc``` privacy practices, please see the privacy policy. You may opt-out of..._ If you set a ```privacyUrl```, this is the text that will be displayed, along with a hyperlink set on *privacy policy*.
```

Comment:
I think it's clearer to say, just before this example: "If you do not set a privacy policy URL, the privacy policy related phrase will be removed from the text."

And then you continue with the example, as you did here.